Sources/Sentry/Public/SentryStacktrace.h

```diff
@@ -7,7 +7,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class SentryFrame;
 
-NS_SWIFT_NAME(Stacktrace)
```

Comment:
`h`: This is a breaking change, and we must wait until the next major to do this.